service/src/main/java/com/ge/predix/acs/privilege/management/dao/ResourceRepositoryProxy.java

```diff
@@ -155,4 +157,12 @@ public ResourceEntity getByZoneAndResourceIdentifier(final ZoneEntity zone, fina
         return this.activeRepository.getByZoneAndResourceIdentifier(zone, resourceIdentifier);
     }
 
+    @Override
+    public Set<String> getResourceEntityAndDescendantsIds(final ResourceEntity entity) {
+        if (this.activeRepository == this.graphRepository) { // i.e. titan is enabled
```

Comment:
Would the code be more readable if we refactored this equality check to a private method called "isGraphEnabled()" or similar?